packages/kit/test/prerendering/basics/src/routes/env/+page.server.js

```diff
@@ -1,9 +1,7 @@
 import { PRIVATE_STATIC } from '$env/static/private';
-import { env } from '$env/dynamic/private';
```

Comment:
This makes me wonder a bit about the `export const prerender = 'auto'` case. How would someone be able to use that on the server? Say someone does this
```js
import { env } from '$env/dynamic/private';

export const prerender = 'auto';

export function load() {
  if (!prerendering) {
    return { x: env.x }
  } else {
   return { x: 'unknown' }
  }
}
```
This would still work with the given code because the proxy does only error if you access one of the properties at runtime, right? Should we have a test for that?